modules/aggregation-layers/src/aggregation-layer-v9/cpu-aggregator/cpu-aggregator.ts

```diff
@@ -0,0 +1,189 @@
+import type {Aggregator, AggregationProps} from '../aggregator';
+import {_deepEqual as deepEqual, BinaryAttribute} from '@deck.gl/core';
+import {sortBins, packBinIds} from './sort';
+import {aggregateChannel} from './aggregate';
+import {VertexAccessor, evaluateVertexAccessor} from './vertex-accessor';
+
+/** Settings used to construct a new CPUAggregator */
+export type CPUAggregatorSettings = {
+  /** Size of bin IDs */
+  dimensions: number;
+  /** Accessor to map each data point to a bin ID.
+   * If dimensions=1, bin ID should be a number;
+   * If dimensions>1, bin ID should be an array with [dimensions] elements;
+   * The data point will be skipped if bin ID is null.
+   */
+  getBin: VertexAccessor<number | number[] | null, any>;
```

Comment:
Nit: Always make it an array for cleaner types? Or there is a big enough convenience in allowing a single number?